components/Header.tsx

```diff
@@ -40,7 +40,7 @@ export const Header: React.FC<HeaderProps> = () => {
           </Link>
           <div className="ml-2 px-2 rounded bg-amber-300">EXPERIMENTAL</div>
         </div>
-        <div className="flex flex-1 gap-4 items-center md:order-1 transform translate-x-1/4">
```

Comment:
I wasn't sure why the `translate-x-1/4` was here, but it was causing horizontal overflow on the document

Reply:
That is there so that on desktop sizes the header search bar is centered horizontally.

Reply:
Thanks for letting me know the motivation. I switched it to only translate on `md`